<ui version="4.0" stdsetdef="1" >
  <author>Petr Vanek &lt;petr@scribus.info></author>
  <comment>This is base class for Scribus Barcode Generator. Always use designer for this file!</comment>
  <exportmacro></exportmacro>
  <class>BarcodeGeneratorBase</class>
  <widget class="QDialog" name="BarcodeGeneratorBase" >
    <property name="geometry" >
      <rect>
        <x>0</x>
        <y>0</y>
        <width>462</width>
        <height>426</height>
      </rect>
    </property>
    <property name="windowTitle" >
      <string>Barcode Creator</string>
    </property>
    <property name="toolTip" stdset="0" >
      <string/>
    </property>
    <layout class="QGridLayout" >
      <item row="0" column="0" >
        <layout class="QVBoxLayout" >
          <property name="margin" >
            <number>0</number>
          </property>
          <item>
            <layout class="QHBoxLayout" >
              <property name="margin" >
                <number>0</number>
              </property>
              <item>
                <widget class="Q3GroupBox" name="bcodeBox" >
                  <property name="title" >
                    <string>Barcode</string>
                  </property>
                  <layout class="QGridLayout" >
                    <property name="margin" >
                      <number>0</number>
                    </property>
                    <item row="0" column="0" >
                      <layout class="QVBoxLayout" >
                        <property name="margin" >
                          <number>0</number>
                        </property>
                        <item>
                          <layout class="QHBoxLayout" >
                            <property name="margin" >
                              <number>0</number>
                            </property>
                            <item>
                              <layout class="QVBoxLayout" >
                                <property name="margin" >
                                  <number>0</number>
                                </property>
                                <item>
                                  <widget class="QLabel" name="bcLabel" >
                                    <property name="text" >
                                      <string>&amp;Type:</string>
                                    </property>
                                    <property name="buddy" stdset="0" >
                                      <cstring>bcCombo</cstring>
                                    </property>
                                    <property name="wordWrap" >
                                      <bool>false</bool>
                                    </property>
                                  </widget>
                                </item>
                                <item>
                                  <widget class="QLabel" name="codeLabel" >
                                    <property name="text" >
                                      <string>Co&amp;de:</string>
                                    </property>
                                    <property name="buddy" stdset="0" >
                                      <cstring>codeEdit</cstring>
                                    </property>
                                    <property name="wordWrap" >
                                      <bool>false</bool>
                                    </property>
                                  </widget>
                                </item>
                              </layout>
                            </item>
                            <item>
                              <layout class="QVBoxLayout" >
                                <property name="margin" >
                                  <number>0</number>
                                </property>
                                <item>
                                  <widget class="QComboBox" name="bcCombo" >
                                    <property name="toolTip" stdset="0" >
                                      <string>Select one of the available barcode type here</string>
                                    </property>
                                  </widget>
                                </item>
                                <item>
                                  <layout class="QHBoxLayout" >
                                    <property name="margin" >
                                      <number>0</number>
                                    </property>
                                    <item>
                                      <widget class="QLineEdit" name="codeEdit" >
                                        <property name="toolTip" stdset="0" >
                                          <string>The numeric representation of the code itself. See the help message below</string>
                                        </property>
                                      </widget>
                                    </item>
                                    <item>
                                      <widget class="QPushButton" name="resetButton" >
                                        <property name="sizePolicy" >
                                          <sizepolicy>
                                            <hsizetype>0</hsizetype>
                                            <vsizetype>0</vsizetype>
                                            <horstretch>0</horstretch>
                                            <verstretch>0</verstretch>
                                          </sizepolicy>
                                        </property>
                                        <property name="maximumSize" >
                                          <size>
                                            <width>24</width>
                                            <height>32767</height>
                                          </size>
                                        </property>
                                        <property name="text" >
                                          <string/>
                                        </property>
                                        <property name="toolTip" stdset="0" >
                                          <string>Reset the barcode samples</string>
                                        </property>
                                      </widget>
                                    </item>
                                  </layout>
                                </item>
                              </layout>
                            </item>
                          </layout>
                        </item>
                        <item>
                          <widget class="QCheckBox" name="textCheck" >
                            <property name="text" >
                              <string>&amp;Include text in barcode</string>
                            </property>
                            <property name="shortcut" >
                              <string>Alt+I</string>
                            </property>
                            <property name="checked" >
                              <bool>true</bool>
                            </property>
                            <property name="toolTip" stdset="0" >
                              <string>If checked, there will be numbers in the barcode too</string>
                            </property>
                          </widget>
                        </item>
                        <item>
                          <widget class="QCheckBox" name="guardCheck" >
                            <property name="text" >
                              <string>&amp;Guard whitespace</string>
                            </property>
                            <property name="shortcut" >
                              <string>Alt+G</string>
                            </property>
                            <property name="toolTip" stdset="0" >
                              <string>Draw arrows to be sure of space next the code</string>
                            </property>
                          </widget>
                        </item>
                        <item>
                          <widget class="QCheckBox" name="includeCheck" >
                            <property name="text" >
                              <string>I&amp;nclude checksum</string>
                            </property>
                            <property name="shortcut" >
                              <string>Alt+N</string>
                            </property>
                            <property name="toolTip" stdset="0" >
                              <string>Generate and include a checksum in barcode</string>
                            </property>
                          </widget>
                        </item>
                        <item>
                          <widget class="QCheckBox" name="includeCheckInText" >
                            <property name="text" >
                              <string>Incl&amp;ude checksum digit</string>
                            </property>
                            <property name="shortcut" >
                              <string>Alt+U</string>
                            </property>
                            <property name="toolTip" stdset="0" >
                              <string>Include the checksum digit in the barcode text</string>
                            </property>
                          </widget>
                        </item>
                      </layout>
                    </item>
                  </layout>
                </widget>
              </item>
              <item>
                <widget class="Q3GroupBox" name="colorBox" >
                  <property name="title" >
                    <string>Colors</string>
                  </property>
                  <layout class="QGridLayout" >
                    <property name="margin" >
                      <number>0</number>
                    </property>
                    <item row="0" column="0" >
                      <layout class="QHBoxLayout" >
                        <property name="margin" >
                          <number>0</number>
                        </property>
                        <item>
                          <layout class="QVBoxLayout" >
                            <property name="margin" >
                              <number>0</number>
                            </property>
                            <item>
                              <widget class="QPushButton" name="bgColorButton" >
                                <property name="text" >
                                  <string>&amp;Background</string>
                                </property>
                                <property name="shortcut" >
                                  <string>Alt+B</string>
                                </property>
                                <property name="toolTip" stdset="0" >
                                  <string>Background color - under the code lines</string>
                                </property>
                              </widget>
                            </item>
                            <item>
                              <widget class="QPushButton" name="lnColorButton" >
                                <property name="text" >
                                  <string>&amp;Lines</string>
                                </property>
                                <property name="shortcut" >
                                  <string>Alt+L</string>
                                </property>
                                <property name="toolTip" stdset="0" >
                                  <string>Color of the lines in barcode</string>
                                </property>
                              </widget>
                            </item>
                            <item>
                              <widget class="QPushButton" name="txtColorButton" >
                                <property name="text" >
                                  <string>&amp;Text</string>
                                </property>
                                <property name="shortcut" >
                                  <string>Alt+T</string>
                                </property>
                                <property name="toolTip" stdset="0" >
                                  <string>Color of the text and numbers</string>
                                </property>
                              </widget>
                            </item>
                          </layout>
                        </item>
                        <item>
                          <layout class="QVBoxLayout" >
                            <property name="margin" >
                              <number>0</number>
                            </property>
                            <item>
                              <widget class="QLabel" name="bgLabel" >
                                <property name="minimumSize" >
                                  <size>
                                    <width>10</width>
                                    <height>0</height>
                                  </size>
                                </property>
                                <property name="frameShape" >
                                  <enum>QFrame::Box</enum>
                                </property>
                                <property name="scaledContents" >
                                  <bool>true</bool>
                                </property>
                                <property name="wordWrap" >
                                  <bool>false</bool>
                                </property>
                              </widget>
                            </item>
                            <item>
                              <widget class="QLabel" name="linesLabel" >
                                <property name="minimumSize" >
                                  <size>
                                    <width>10</width>
                                    <height>0</height>
                                  </size>
                                </property>
                                <property name="frameShape" >
                                  <enum>QFrame::Box</enum>
                                </property>
                                <property name="scaledContents" >
                                  <bool>true</bool>
                                </property>
                                <property name="wordWrap" >
                                  <bool>false</bool>
                                </property>
                              </widget>
                            </item>
                            <item>
                              <widget class="QLabel" name="txtLabel" >
                                <property name="minimumSize" >
                                  <size>
                                    <width>10</width>
                                    <height>0</height>
                                  </size>
                                </property>
                                <property name="frameShape" >
                                  <enum>QFrame::Box</enum>
                                </property>
                                <property name="scaledContents" >
                                  <bool>true</bool>
                                </property>
                                <property name="wordWrap" >
                                  <bool>false</bool>
                                </property>
                              </widget>
                            </item>
                          </layout>
                        </item>
                      </layout>
                    </item>
                  </layout>
                </widget>
              </item>
            </layout>
          </item>
          <item>
            <layout class="QHBoxLayout" >
              <property name="margin" >
                <number>0</number>
              </property>
              <item>
                <widget class="Q3TextEdit" name="commentEdit" >
                  <property name="readOnly" >
                    <bool>true</bool>
                  </property>
                  <property name="toolTip" stdset="0" >
                    <string>Hints and help is shown here</string>
                  </property>
                </widget>
              </item>
              <item>
                <widget class="QLabel" name="sampleLabel" >
                  <property name="frameShape" >
                    <enum>QFrame::Box</enum>
                  </property>
                  <property name="alignment" >
                    <set>Qt::AlignCenter</set>
                  </property>
                  <property name="toolTip" stdset="0" >
                    <string>Preview of the result. 72dpi sample.</string>
                  </property>
                  <property name="wordWrap" >
                    <bool>false</bool>
                  </property>
                </widget>
              </item>
            </layout>
          </item>
          <item>
            <layout class="QHBoxLayout" >
              <property name="margin" >
                <number>0</number>
              </property>
              <item>
                <spacer name="spacer2" >
                  <property name="sizeHint" >
                    <size>
                      <width>40</width>
                      <height>20</height>
                    </size>
                  </property>
                  <property name="sizeType" >
                    <enum>Expanding</enum>
                  </property>
                  <property name="orientation" >
                    <enum>Horizontal</enum>
                  </property>
                </spacer>
              </item>
              <item>
                <widget class="QPushButton" name="okButton" >
                  <property name="text" >
                    <string/>
                  </property>
                </widget>
              </item>
              <item>
                <widget class="QPushButton" name="cancelButton" >
                  <property name="text" >
                    <string/>
                  </property>
                </widget>
              </item>
            </layout>
          </item>
        </layout>
      </item>
    </layout>
  </widget>
  <layoutdefault spacing="6" margin="11" />
  <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
  <connections>
    <connection>
      <sender>bcCombo</sender>
      <signal>activated(QString)</signal>
      <receiver>BarcodeGeneratorBase</receiver>
      <slot>bcComboChanged()</slot>
    </connection>
    <connection>
      <sender>textCheck</sender>
      <signal>stateChanged(int)</signal>
      <receiver>BarcodeGeneratorBase</receiver>
      <slot>textCheck_changed()</slot>
    </connection>
    <connection>
      <sender>guardCheck</sender>
      <signal>stateChanged(int)</signal>
      <receiver>BarcodeGeneratorBase</receiver>
      <slot>guardCheck_changed()</slot>
    </connection>
    <connection>
      <sender>bgColorButton</sender>
      <signal>clicked()</signal>
      <receiver>BarcodeGeneratorBase</receiver>
      <slot>bgColorButton_pressed()</slot>
    </connection>
    <connection>
      <sender>lnColorButton</sender>
      <signal>clicked()</signal>
      <receiver>BarcodeGeneratorBase</receiver>
      <slot>lnColorButton_pressed()</slot>
    </connection>
    <connection>
      <sender>txtColorButton</sender>
      <signal>clicked()</signal>
      <receiver>BarcodeGeneratorBase</receiver>
      <slot>txtColorButton_pressed()</slot>
    </connection>
    <connection>
      <sender>okButton</sender>
      <signal>clicked()</signal>
      <receiver>BarcodeGeneratorBase</receiver>
      <slot>okButton_pressed()</slot>
    </connection>
    <connection>
      <sender>cancelButton</sender>
      <signal>clicked()</signal>
      <receiver>BarcodeGeneratorBase</receiver>
      <slot>cancelButton_pressed()</slot>
    </connection>
    <connection>
      <sender>codeEdit</sender>
      <signal>textChanged(QString)</signal>
      <receiver>BarcodeGeneratorBase</receiver>
      <slot>codeEdit_textChanged(QString)</slot>
    </connection>
    <connection>
      <sender>resetButton</sender>
      <signal>clicked()</signal>
      <receiver>BarcodeGeneratorBase</receiver>
      <slot>resetButton_clicked()</slot>
    </connection>
    <connection>
      <sender>includeCheck</sender>
      <signal>stateChanged(int)</signal>
      <receiver>BarcodeGeneratorBase</receiver>
      <slot>includeCheck_stateChanged(int)</slot>
    </connection>
    <connection>
      <sender>includeCheckInText</sender>
      <signal>stateChanged(int)</signal>
      <receiver>BarcodeGeneratorBase</receiver>
      <slot>includeCheckInText_stateChanged(int)</slot>
    </connection>
  </connections>
</ui>
